app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,4 @@
 // Specific CSS for your home-page
+.container {
```

Comment:
careful!
this is a bootstrap class. 
it's better to rename the class to something else or use bootstrap all the way
@Leprof20 @Sajakhtar

Reply:
Thanks, fixed!